packages/core/useBrowserLocation/index.ts

```diff
@@ -27,6 +27,8 @@ export interface BrowserLocationState {
  * @param options
  */
 export function useBrowserLocation({ window = defaultWindow }: ConfigurableWindow = {}) {
+  if (getCurrentInstance()?.appContext.config.globalProperties.$router) throw new Error('Do not use `useBrowserLocation` with Vue Router installed, use `useRoute` provided by Vue Router instead.')
```

Comment:
In VueUse we are trying to avoid throw runtime errors. Also, since we can't imagine what's the usage that ppl needs, we shouldn't babysit here. I think a line in the doc is enough.